vignettes/igraph_ES.rmd

```diff
@@ -0,0 +1,640 @@
+---
+title: "igraph (R interface) in spanish"
+date: "`r Sys.Date()`"
```

Comment:
I'd suggest removing the date as it's not useful information.

In https://r-pkgs.org/vignettes.html

> date: we think this usually does more harm than good, since it’s not clear what the date is meant to convey. Is it the last time the vignette source was updated? In that case you’ll have to manage it manually and it’s easy to forget to update it. If you manage date programmatically with Sys.date(), the date reflects when the vignette was built, i.e. when the package bundle was created, which has nothing to do with when the vignette or package was last modified. We’ve decided it’s best to omit the date.

Reply:
I've made a PR but I'm now realizing I should have suggested a line change directly via the PR interface (I don't have write access to this repo), sorry for the clunkiness.